projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -94,7 +94,6 @@ export function selectAccountPayment() {
   );
   cy.get('button.btn-primary').click({ force: true });
   cy.wait(`@${shippingPage}`).its('response.statusCode').should('eq', 200);
-  cy.wait('@getCart').its('response.statusCode').should('eq', 200);
```

Comment:
🤔 
maybe not haha :P because line 84-89 is still there waiting to intercept ;P

Let's have a look on what's happening and if there's flakiness

Reply:
I might be wrong, but I vaguely remember Marcin and I removed it because we no longer do the cart call?